docs/UserGuide.md

```diff
@@ -196,9 +203,9 @@ Examples:
 * `find e/yahoo` returns Alex Yeoh and David Li (both uses yahoo email)
 * `find h/john` returns John Doe and John Lee (telegram handle @johnd and @johnlee respectively)
 * `find t/tutorial-1 t/high-ability` returns Samantha (is tagged with BOTH tutorial-1 and high-ability)
-* `find n/john t/tutorial-1` returns John Doe (name contains john and is tagged with tutorial-1)
+* `find n/john t/tutorial-2` returns John Doe (name contains john and is tagged with tutorial-1)
```

Comment:
Is this a typo? I think the command was meant to return the students whose name contains john and is tagged with **tutorial-2**.

Reply:
Yes thx for pointing this out, i forgot to change the second "tutorial-1"